Directory.Build.props

```diff
@@ -51,16 +51,16 @@
     <Copyright>Copyright � John Kelly and Contributors</Copyright>
     <Description>A high speed vector maths library</Description>
     <Features>strict</Features>
-    <!--<GenerateDocumentationFile>true</GenerateDocumentationFile>-->
-    <AssemblyOriginatorKeyFile>$(MSBuildThisFileDirectory)MathSharp.snk</AssemblyOriginatorKeyFile>
+    <GenerateDocumentationFile>false</GenerateDocumentationFile>
     <HighEntropyVA>true</HighEntropyVA>
     <LangVersion>preview</LangVersion>
+    <MathSharpPublicKey>00240000048000009400000006020000002400005253413100040000010001000586cd22205c2991de00dea27c3455744874d5b9c4532ccee2bf8fa9b55752af42fa6170f641c20d1501cd3b9e6d22a1327cdaf19eb8eba23e8ed350a60930e7286417916e3a3a39798e69d8d521ac3af4c720cbcca2f2fbf5cee2c0b1c6be08ebba9051399148c663c3408513be9a2ff96e7816bb1b3713d1f30e13fa2e66a9</MathSharpPublicKey>
```

Comment:
It's generally good to have your own SNK file as well. It isn't private, but it is used for "identity".

The one from TerraFX is meant for TerraFX 😛. I generated a new one for you as part of this PR.

Reply:
(you can also generate your own using the VS UX, when needed)